tests/nnstreamer_sink/unittest_sink.cpp

```diff
@@ -866,7 +866,10 @@ _get_temp_filename (void)
   }
 
   g_close (fd, NULL);
-  g_remove (tmp_fn);
+  if (g_remove (tmp_fn) != 0) {
+    _print_log ("failed to remove temp file %s", tmp_fn);
```

Comment:
If this is removed in ``` _get_temp_filename() ```, how can the users use the temp file?

Reply:
It test is failed (assertion case), this file will not be removed.